scripts/enable-backend.sh

```
@@ -21,20 +24,13 @@
 # setup
 ##############################
 
-baseDirectory="/var/docker"
+scriptDir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+baseDirectory="$(cd "$scriptDir/../.." && pwd)"   # parent of the ndb-setup checkout (instances live here)
 source "$baseDirectory/ndb-setup/setup.env"
```

Shellcheck (0.11.0)

[info] Not following: ./ndb-setup/setup.env was not specified as input (see shellcheck -x).

(SC1091)